### DISCUSSION TITLE
Poetry garbage output when running in Jenkins and CI/CD system

### DISCUSSION THREAD

**Discussion by vit1251** [python-poetry/poetry#5792] [Q&A]
During verbose mode Poetry execution CI/CD output is poisoned by repeatable messages about package installation progress.

I assume that uses cursor move operation in sequential stream CI/CD output. How to disable cursor movement operations and make output more pretty.

**Comment by mkniewallner** [comment]
Does passing `--no-ansi` option to Poetry commands helps?

**Reply by vit1251** [reply]
No output is look like one blob. You can see below
![image](https://user-images.githubusercontent.com/110534/178801889-bf364f7a-a380-48b5-bf97-969f8502cadc.png)

**Reply by mkniewallner** [reply]
Agree that this is not perfect, but it still seems better than having repeatable blocks.
What version of Poetry are you using, out of curiosity?

**Reply by vit1251** [reply]
Look like I use an poetry version 1.1.14

**Reply by jimmywan** [reply]
My GH Actions output does not look like that (running poetry as part of a docker image build):

```bash
[844]Step 25/36 : RUN poetry install -vvv --no-dev
 ---> Running in XXXXXXXXXXXX
Creating virtualenv object-service in /app/.venv
Using virtualenv: /app/.venv
Installing dependencies from lock file

Finding the necessary packages for the current system

Package operations: 78 installs, 0 updates, 0 removals, 12 skipped

  • Removing atomicwrites (1.4.1): Skipped for the following reason: Not currently installed
  • Removing black (22.6.0): Skipped for the following reason: Not currently installed
  • Removing flake8 (4.0.1): Skipped for the following reason: Not currently installed
  • Removing isort (5.10.1): Skipped for the following reason: Not currently installed
...
  • Installing protobuf (3.20.1)
  • Installing certifi (2022.6.15)
  • Installing urllib3 (1.26.11)
...
```
